operators/derive/to_mapbox.py

```diff
@@ -0,0 +1,99 @@
+import dataflows as DF
+from dataflows_airtable import load_from_airtable
+
+from conf import settings
+from operators.mapbox_upload import upload_tileset
+from srm_tools.logger import logger
+
+from . import helpers
+
+
+def geo_data_flow():
+    return DF.Flow(
+        DF.load(f'{settings.DATA_DUMP_DIR}/table_data/datapackage.json'),
+        DF.update_package(name='Geo Data'),
+        DF.update_resource(['table_data'], name='geo_data', path='geo_data.csv'),
+        DF.join_with_self(
+            'geo_data',
+            ['response_id', 'branch_geometry'],
+            fields=dict(
+                branch_geometry=None,
+                response_id=None,
+                response_name=None,
+                organization_id={'name': 'organization_id', 'aggregate': 'set'},
+                organization_name={'name': 'organization_name', 'aggregate': 'set'},
+                branch_id={'name': 'branch_id', 'aggregate': 'set'},
+                branch_name={'name': 'branch_name', 'aggregate': 'set'},
+                service_id={'name': 'service_id', 'aggregate': 'set'},
+                service_name={'name': 'service_name', 'aggregate': 'set'},
+                situation_id={'name': 'situation_id', 'aggregate': 'set'},
+                situation_name={'name': 'situation_name', 'aggregate': 'set'},
+            ),
+        ),
+        DF.add_field(
+            'organizations',
+            'array',
+            lambda r: [
+                {'id': vals[0], 'name': vals[1]}
+                for vals in zip(r['organization_id'], r['organization_name'])
+            ],
+        ),
+        DF.add_field(
+            'branches',
+            'array',
+            lambda r: [
+                {'id': vals[0], 'name': vals[1]} for vals in zip(r['branch_id'], r['branch_name'])
```

Comment:
aggregating by set doesn't necessarily preserve order, which means this is a bit risky

Reply:
yep.

Reply:
yes.